static/js/components/UserInfoCard.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react';
+import { Card } from 'react-mdl/lib/Card';
+import Icon from 'react-mdl/lib/Icon';
+import IconButton from 'react-mdl/lib/IconButton';
+
+import ProfileImage from './ProfileImage';
+import {
+  getEmployer,
+  getPreferredName,
+  userPrivilegeCheck
+} from '../util/util';
+import { mstr } from '../util/sanctuary';
+import type { Profile } from '../flow/profileTypes';
+
+export default class UserInfoCard extends React.Component {
+  props: {
+    profile: Profile,
+    toggleShowPersonalDialog: () => void
+  };
+
+  render() {
+    const { profile, toggleShowPersonalDialog } = this.props;
+
+    return (
+      <Card shadow={1} className="profile-form user-page">
+        <div className="user-form-grid">
```

Comment:
it seems like you should be able to re-use `.profile-form-row` here, and, if you refactor the column class along the lines I suggested above, you should also be able to re-use the `.col` class, then you could make the class for this `.profile-form-row.user-card`, and use that selector to override any styles you needed to on the user card (e.g. for font sizes)